lib/support/clone_vm.rb

```diff
@@ -428,12 +428,83 @@ def find_datacenter
       root_folder.childEntity.grep(RbVmomi::VIM::Datacenter).find { |x| x.name == datacenter }
     end
 
+    def customization_spec
+      unless options[:guest_customization]
```

Comment:
I tested this branch against our vCenter installation, and I found there is a missing line - you need to pass the config into the options. Add something like:
```
guest_customization: config[:guest_customization],
```
near here https://github.com/chef/kitchen-vcenter/blob/8a7b668ef588b38653df13dd06e8b6ccfda4aad6/lib/kitchen/driver/vcenter.rb#L161

Without that, this line is always false, and the customizations do not get applied.

Reply:
@clintoncwolfe sorry about that, and you are exactly right. That line did not get pushed. I will be pushing with these changes and others shortly.